apps/electronlauncher/electronlauncher.yml

```diff
@@ -0,0 +1,10 @@
+name: Electron Launcher
```

Comment:
As per https://github.com/electron/apps/blob/master/contributing.md#company-logos-and-names --

 If GitHub has given you permission to use that name, please provide that information in the PR so that the app's name can be used as-is. Otherwise,

 > Please do not directly use another company's name or product without permission. It's generally better to refer to it in a dependent clause; for example, after "compatible with", "on", or "for."
>
> For example, while we would not accept a third-party app named "GitHub Notifications", we would consider "Yourname Notifications for GitHub".

Reply:
Thanks, I'll work on a different name when I get time and update the repo and PR.